style-guide/tertiary.html

```diff
@@ -131,14 +131,14 @@
 										<li id="menu-item-129"><a href="clinical-partners/partner-with-us/">Partner With Us</a></li>
 									</ul>
 								</li>
-								<li id="menu-item-120"><a href="about/">About</a>
+								<li id="menu-item-513" class="non-anchor parent"><span class="parent-anchor">About</span>
```

Comment:
This looks like a copy/paste from developer tools rather than "view source". The `<span>` isn't added until JavaScript processes it. This should be changed to match expected output.